### PR TITLE
Allow `-sFOO=1` as well as `-s FOO=1` on the command line

### PR DESCRIPTION
**Pull request by sbc100**
Fixes: #11463

### CODE REVIEW

**Comment by kripken** [comment]
Oh, also the `emcc --help`  docs for `-s` as well.

**Comment by sbc100** [comment]
Done

File: docs/emcc.txt [hunk 1]
```diff
@@ -113,6 +113,9 @@ Options that are modified or new in *emcc* are listed below:
 
      * The specified file path must be absolute, not relative.
 
+   Note: Options can be specified as a single argument without a
+     space between the "-s" and option name.  .e.g "-sFOO=1".
```
Comment (kripken) [inline docs/emcc.txt:117]:
```suggestion
     space between the "-s" and option name, e.g., "-sFOO=1".
```

File: site/source/docs/tools_reference/emcc.rst [hunk 2]
```diff
@@ -106,6 +106,9 @@ Options that are modified or new in *emcc* are listed below:
     - In this case the file might contain a JSON-formatted list of functions: ``["_func1", "func2"]``.
     - The specified file path must be absolute, not relative.
 
+  .. note:: Options can be specified as a single argument without a space
+            between the ``-s`` and option name.  .e.g ``-sFOO=1``.
```
Comment (kripken) [inline site/source/docs/tools_reference/emcc.rst:110]:
```suggestion
            between the ``-s`` and option name, e.g., ``-sFOO=1``.
```

**Pull request merged**